weeder.toml

```diff
@@ -0,0 +1,10 @@
+roots = [
+    "^Main.main$",
```

Comment:
I think this should get all our executables and their dependencies, but it's probably buggy. 😕